cupyx/linalg/linalg.py

```diff
@@ -0,0 +1,99 @@
+import numpy
+
+import cupy
+from cupy import cuda
+from cupy.cuda import cublas
+from cupy.cuda import device
+from cupy.linalg import util
+
+if cuda.cusolver_enabled:
+    from cupy.cuda import cusolver
+
+
+def invh(a):
+    """Compute the inverse of a Hermitian matrix.
+
+    This function computes a inverse of a real symmetric or complex hermitian
+    matrix.
+
+    Args:
+        a (cupy.ndarray): Real symmetric or complex hermitian maxtix.
+
+    Returns:
+        cupy.ndarray: The inverse of matrix ``a``.
+    """
+
+    if not cuda.cusolver_enabled:
+        raise RuntimeError('cuSOLVER is not available')
+
+    # to prevent `a` to be overwritten
```

Comment:
A nit. Although I noticed this already exists in other places.
```suggestion
    # to prevent `a` from being overwritten
```

Reply:
Thanks, I did it.